app/config/services.yml

```diff
@@ -14,12 +14,22 @@ services:
         # if you need to do this, you can override this setting on individual services
         public: false
 
-    # loads services from whatever directories you want (you can add directories!)
+    # makes all classes in src/AppBundle available as services
     # this creates a service per class whose id is the fully-qualified class name
     AppBundle\:
-        resource: '../../src/AppBundle/{Command,Form,EventSubscriber,Twig,Security}'
+        resource: '../../src/AppBundle/*'
+        # you can include directories or files
+        # but if a service is unused, it's removed anyways
+        exclude: '^(AppBundle\.php|Entity|Repository)'
 
+    # controllers are imported separately to make sure they're public
+    # and have a tag that makes allows actions to type-hint services
     AppBundle\Controller\:
         resource: '../../src/AppBundle/Controller'
         public: true
         tags: ['controller.service_arguments']
+
+    # add more service, or override services that need manual wiring
```

Comment:
Ha, thank you for the review! I managed to make a lot of typos... in a very small area :).